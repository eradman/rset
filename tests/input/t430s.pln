# a comment

interpreter=/bin/ksh -x
local_interpreter=/bin/sh -e

common packages:
{
	echo "MIRROR=http://cdn.openbsd.org/pub/OpenBSD"
}
	pkg_add rsync-- ruby%2.6
	echo "inet 172.16.0.1/16" > /etc/hostname.vether0
	echo "add vether0" > /etc/hostname.bridge0
	
	cat <<-EOF > /dev/null
	fake label:
	# comment
	EOF

desktop:
	pkg_add hermit-font vim--gtk2

# vim:noexpandtab:syntax=sh:ts=4
