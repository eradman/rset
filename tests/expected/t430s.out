[0] common packages
MIRROR=http://cdn.openbsd.org/pub/OpenBSD
pkg_add rsync-- ruby%2.6
echo "inet 172.16.0.1/16" > /etc/hostname.vether0
echo "add vether0" > /etc/hostname.bridge0

cat <<-EOF > /dev/null
fake label:
# comment
EOF
.
[1] desktop
pkg_add hermit-font vim--gtk2
.
